### PR TITLE
Remove calico/node from this repo

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description

It now lives in https://github.com/projectcalico/node

Updates the release procedure and Makefile as well. 

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,88 +1,259 @@
...
+   - [networking-calico](https://github.com/projectcalico/networking-calico)
+
+   The following components do not share a version with the Calico release, but are included in the documentation.
+
+   - [flannel](https://github.com/coreos/flannel/releases)
+
+## Preparing to cut a Calico release
+
+
+1. Check out the `master` branch of this repository and make sure it is up-to-date and passing Semaphore CI.
```
Comment (ozdanborne) [inline RELEASING.md]: a link to semaphore's CI for calico master would be nifty: https://semaphoreci.com/calico/calico/branches/master

File: RELEASING.md [hunk 2]
```diff
@@ -1,88 +1,259 @@
...
+   ```
+   git checkout master && git pull origin master
+   ```
+
+Your next steps depend on the type of release:
+- [Creating a new major/minor release](#major-minor)
+- [Promoting a release candidate](#promoting)
+- [Creating a patch release](#patch)
+
+## <a name="major-minor"></a> Creating a new major / minor release
```
Comment (ozdanborne) [inline RELEASING.md]: what's up with this `<a>` tag

File: RELEASING.md [hunk 3]
```diff
@@ -1,88 +1,259 @@
...
+
+   ```
+   git commit -m "Updates for release vX.Y"
+   ```
+
+1. Run the following on your local branch in order to build and publish the release
+   at the newly created commit.
+
+   ```
+   make VERSION=vX.Y.Z TYPHA_VER=<TYPHA_VER> release
```
Comment (ozdanborne) [inline RELEASING.md]: would `VERSION=vX.Y.Z-rcN` for a release candidate?

File: RELEASING.md (hunk 2 above)
Comment (caseydavenport) [reply RELEASING.md]: It's an anchor to this heading for linking so we don't need to do #creating-a-new-major-minor-release and update it whenever the title changes.

**Comment by caseydavenport** [comment]
> I note that you're removing some calico/node components but not all (e.g. felix). Is that because they can be installed outside of calico/node?

Yeah, I'll double check but I believe we still document how to install Felix on its own.

If we're not using that version info then I'd love to remove it, but I suspect we still are.

**Comment by caseydavenport** [comment]
Looks like Felix _wasn't_ used in the docs.  Hooray!

**Pull request merged**